.env.dist

```diff
@@ -48,3 +48,6 @@ SECRET_VIDEORES_FTP_USERNAME=
 SECRET_VIDEORES_FTP_PWD=
 SECRET_VIDEORES_FILEPATH=
 VIDEORES_HASH_CLIENT_IP=true
+
+# InstructureUI
+DISABLE_SPEEDY_STYLESHEET=false
```

Comment:
@jackvanb This is always needed right? If so, then hardcode the value in webpack.config.js

Reply:
@jackvanb What does this setting do? I cannot find it documented anywhere

Reply:
Looks like it is part of the glamor css (https://github.com/threepointone/glamor#speedy-mode). It's defined in Instructure ui-stylesheet (https://github.com/instructure/instructure-ui/blob/master/packages/ui-stylesheet/src/StyleSheet.js#L30).

Reply:
So DISABLE_SPEEDY_STYLESHEET needs to be in .env, because it reads from process.env.

Reply:
@jackvanb Is this setting needed?

I checked out your branch and ran yarn and then yarn dev without editing my .env file and the web app still seems to work.